citylearn/data/citylearn_2022_phase_all_plus_evs/schema.json

```diff
@@ -0,0 +1,1034 @@
+      }
+    },
+    "EV_5": {
+      "include": true,
+      "energy_simulation": "EV_5.csv",
+      "inactive_observations": [],
+      "inactive_actions": [],
+      "image_path": "images/ev.png",
+      "battery": {
+        "type": "citylearn.energy_model.Battery",
+        "autosize": false,
+        "attributes": {
+          "capacity": 90,
+          "nominal_power": 50,
+          "initial_soc": 0.8
+        }
+      }
+    },
+    "EV_6": {
+      "include": true,
+      "energy_simulation": "EV_6.csv",
+      "inactive_observations": [],
+      "inactive_actions": [],
+      "image_path": "images/ev.png",
+      "battery": {
+        "type": "citylearn.energy_model.Battery",
+        "autosize": false,
+        "attributes": {
+          "capacity": 100,
+          "nominal_power": 50,
+          "initial_soc": 0.8
+        }
+      }
+    },
+    "EV_7": {
+      "include": true,
+      "energy_simulation": "EV_7.csv",
+      "inactive_observations": [],
+      "inactive_actions": [],
+      "image_path": "images/ev.png",
+      "battery": {
+        "type": "citylearn.energy_model.Battery",
+        "autosize": false,
+        "attributes": {
+          "capacity": 60,
+          "nominal_power": 50,
+          "initial_soc": 0.1
+        }
+      }
+    },
+    "EV_8": {
+      "include": true,
+      "energy_simulation": "EV_8.csv",
+      "inactive_observations": [],
+      "inactive_actions": [],
+      "image_path": "images/ev.png",
+      "battery": {
+        "type": "citylearn.energy_model.Battery",
+        "autosize": false,
+        "attributes": {
+          "capacity": 80,
+          "nominal_power": 50,
+          "initial_soc": 0.2
+        }
+      }
+    }
+  },
+  "buildings": {
+    "Building_1": {
+      "include": true,
+      "energy_simulation": "Building_1.csv",
+      "weather": "weather.csv",
+      "carbon_intensity": "carbon_intensity.csv",
+      "pricing": "pricing.csv",
+      "coordinates": {
+        "latitude": 41.195632,
+        "longitude": -8.598365
+      },
+      "image_path": "images/building.png",
+      "reward_type": "D",
+      "inactive_observations": [],
+      "inactive_actions": [],
+      "electrical_storage": {
+        "type": "citylearn.energy_model.Battery",
+        "autosize": false,
+        "attributes": {
+          "capacity": 6.4,
+          "efficiency": 0.9,
+          "capacity_loss_coefficient": 1e-05,
+          "loss_coefficient": 0.0,
+          "nominal_power": 5.0
+        }
+      },
+      "pv": {
+        "type": "citylearn.energy_model.PV",
+        "autosize": false,
+        "attributes": {
+          "nominal_power": 12.0
+        }
+      },
+      "chargers": {
+        "charger_1_1": {
+          "type": "citylearn.electric_vehicle_charger.Charger",
+          "autosize": false,
+          "image_path": "images/charger.png",
+          "attributes": {
+            "nominal_power": 11,
+            "efficiency": 0.95,
+            "charger_type": 0,
```

Comment:
Chareger_type refers to whether it is a public or a private charger (explained in electric_vehicle_charger.py file). I am leaving it as it is because it is later used to define the charger_class_name.